test/integration/cdn/test_application.rb

```diff
@@ -10,7 +10,7 @@
     @hanami_env       = ENV['HANAMI_ENV']
     ENV['HANAMI_ENV'] = 'production'
 
-    assert system("HANAMI_ENV=production bundle exec hanami assets precompile")
+    system("HANAMI_ENV=production bundle exec hanami assets precompile")
```

Comment:
This is a working in progress, I'll review everything before I submit for review.

In this specific case, looks like the command is successfully executed but the return is nil. I'm currently investigating it and it might be a fix I have to do in JRuby side.

Thanks for bringing this up.